byok/registries.conf

```diff
@@ -0,0 +1,3 @@
+[[registry]]
```

Comment:
what is this registry used for?

Reply:
That's was a workaround for quay being down, I've removed it. Thanks for catching!